src/components/dialogs/network-modifications-dialog.js

```diff
@@ -52,19 +54,25 @@ const NetworkModificationDialog = ({
 
     const closeDialog = () => {
         setDialogOpen(undefined);
+        setEditData(undefined);
     };
 
     const openDialog = (dialogId) => {
         setDialogOpen(dialogId);
     };
 
+    useEffect(() => {
+        setDialogOpen(editData?.type);
```

Comment:
dialogs are now directly created on network-modification-node-list (the highest component) and are given to network-modifications to avoid duplicating dialogs